objectivec/include/ort_session.h

```diff
@@ -16,15 +20,17 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)init NS_UNAVAILABLE;
 
 /**
- * Creates an ORT Session.
+ * Creates a session.
  *
  * @param env The ORT Environment instance.
  * @param path The path to the ONNX model.
+ * @param sessionOptions Session configuration options.
  * @param[out] error Optional error information set if an error occurs.
  * @return The instance, or nil if an error occurs.
  */
 - (nullable instancetype)initWithEnv:(ORTEnv*)env
                            modelPath:(NSString*)path
+                      sessionOptions:(ORTSessionOptions*)sessionOptions
                                error:(NSError**)error NS_DESIGNATED_INITIALIZER;
```

Comment:
Is there a way to denote that a parameter is optional (assuming it is here)?

Reply:
this is actually not optional now (it's between the NS_ASSUME_NONNULL_BEGIN/END macros). it's not a bad idea to make it optional though (using `nullable`)